src/styles/card.css

```
@@ -0,0 +1,110 @@
+/* #region card */
+:root {
+  font-family: Roboto, sans-serif;
+
+  --secondary-text-color: #616070;
+  --button-color: #00acdc;
+  --main-border-radius: 5px;
+  --text-color: #060b35;
+}
+
+body,
+h2,
+p,
+div {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 198px;
+  border: solid #f3f3f3 1px;
+  border-radius: var(--main-border-radius);
+}
+
+.card__imac-img {
+  height: 134px;
+  margin-top: 32px;
+}
+
+.card__name {
+  width: 166px;
+  margin-top: 40px;
+  color: var(--text-color);
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__code {
+  margin-top: 4px;
+  margin-right: auto;
+  padding-left: 16px;
```

only margin-top is needed
```suggestion
  margin-top: 4px;
```